docs/source/index.rst

```diff
@@ -89,6 +89,12 @@ Contents
    apidocs/qam
    apidocs/devices
 
+.. toctree::
```

Comment:
might as well give this a `:caption:` like "man pages" or something

Reply:
okay, sure :+1: